client/api.c

```diff
@@ -469,7 +469,7 @@ TDNFClean(
 }
 
 //Show count of installed
-//not a dnf command just sanity check
+//not a dnf command just confidence check
```

Comment:
I think we can use `// not a tdnf command, simple validation` instead

Reply:
resolved